### PR TITLE
Create RobGibbens

### PR DESCRIPTION
**Pull request by RobGibbens**

### CODE REVIEW

**Comment by RobGibbens** [comment]
Added RobGibbens.cs to csproj

**Comment by ghuntley** [comment]
One more thing; build failed because `usings` are missing.  See https://raw.githubusercontent.com/planetxamarin/planetxamarin/master/src/Firehose.Web/Authors/JamesMontemagno.cs

```
using Firehose.Web.Infrastructure;
using System;
using System.Collections.Generic;
```

**Comment by RobGibbens** [comment]
Added using statements.  That's what I get for trying to edit the file in the Github web UI and following the directions in the README. :(

File: src/Firehose.Web/Authors/RobGibbens.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+public class RobGibbens : IWorkAtXamarinOrMicrosoft
+{
+    public string FirstName => "Rob";
+    public string LastName => "Gibbens";
+    public string Title => "Manager / Trainer @ Xamarin University";
+    public string StateOrRegion => "Farmington, Michigan";
+    public string EmailAddress => "RobGibbens@arteksoftware.com";
+    public string TwitterHandle => "RobGibbens";
+    public string GravatarHash => "";
+    public DateTime Started => new DateTime(2004, 01, 06);
```
Comment (jamesmontemagno) [inline src/Firehose.Web/Authors/RobGibbens.cs]: I think you mean 2014?

File: src/Firehose.Web/Authors/RobGibbens.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+public class RobGibbens : IWorkAtXamarinOrMicrosoft
+{
+    public string FirstName => "Rob";
+    public string LastName => "Gibbens";
+    public string Title => "Manager / Trainer @ Xamarin University";
+    public string StateOrRegion => "Farmington, Michigan";
+    public string EmailAddress => "RobGibbens@arteksoftware.com";
+    public string TwitterHandle => "RobGibbens";
+    public string GravatarHash => "";
```
Comment (jamesmontemagno) [inline src/Firehose.Web/Authors/RobGibbens.cs]: Please add our hash: 6c67ac9df1543aa59cd14fa52882b7b6

**Comment by jamesmontemagno** [comment]
Woops... might need some more updates on this at some point @RobGibbens

**Pull request merged**